content/html/concepts/attributes/style/style.md

```diff
@@ -0,0 +1,38 @@
+---
+Title: 'Style'
+Description: 'The style attribute specifies how a page will be rendered in a web browser.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Style' 
+  - 'Attributes'
+  - 'Elements'
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `style` attribute specifies how an element will be rendered in a web browser, e.g. the color or the size of a font in a paragraph.
+
+When we use the `style` attribute inline, it overrides any styles set globally. 
+
+**Note:** We can also specify the style of our elements through an external CSS style sheet or embedded in the `<head>` element using the `<style>` tag.
+
+## Syntax
+
+```pseudo
+<element_tag style="property1:value;property2:value;"></element_tag>
+```
+The `style` attribute supports all HTML tags and can take many properties, e.g. you can use the `style` attribute in the `<p>`, the `<h1>` or the `<hr>`.
```

Comment:
```suggestion

The `style` attribute supports all HTML tags and can take many properties, e.g., the `style` attribute can be used in the `<p>`, the `<h1>` or the `<hr>`.
```

Reply:
I'm thinking we can set a standard here for the rest of the global attribute term entries. This is a trait that all global attributes have in that they support all HTML tags. Can we include the keyword "global attribute"

Reply:
done